src/config.js

```diff
@@ -54,13 +54,15 @@ function createGlobalConfig (overrides) {
     const isMobileApp = ['ios', 'android'].includes(userPreferences?.platform.name) || isAndroid
     const isFirefox = navigator.userAgent.includes('Firefox')
     const isDDGDomain = Boolean(window.location.href.match(DDG_DOMAIN_REGEX))
+    const isExtension = Boolean(window.chrome) || isFirefox
```

Comment:
The `chrome` object is not in `window` on Firefox. Let me know if you can think of a better solution.

Reply:
This has the potential to cause confusion, because `window.chrome` exists in our Windows browser too. I'd remove this `isExtension` and rely on the existing logic in DeviceInterface for now. It's far from ideal - and I know of ways to solve it, but for the sake of moving this PR forward I would just `device instanceOf ExtensionInterface` instead :)

Reply:
Great suggestion! Thank you.

Reply:
I ended up setting the `platform` from the transport 🎉.